lib/client/config.go

```diff
@@ -254,15 +256,65 @@ func (c *config) GetNodeLogLevel(node string) (string, ConfigLocation, error) {
 }
 
 // GetFelixConfig provides a mechanism for getting arbitrary Felix configuration
-// in the datastore.
-func (c *config) GetFelixConfig(name string) (*string, error) {
-	return c.getValue(model.GlobalConfigKey{Name: name})
+// in the datastore.  An empty value for the node will get the global
+// configuration, a non-empty node will get the per-node configuration.
+func (c *config) GetFelixConfig(name, node string) (*string, error) {
+	if node == "" {
+		return c.getValue(model.GlobalConfigKey{Name: name})
+	} else {
+		return c.getValue(model.HostConfigKey{Hostname: node, Name: name})
+	}
 }
 
 // SetFelixConfig provides a mechanism for setting arbitrary Felix configuration
 // in the datastore.  A nil value will remove the entry from the datastore.
-func (c *config) SetFelixConfig(name string, value *string) error {
-	key := model.GlobalConfigKey{Name: name}
+// An empty value for the node will set the global configuration, a non-empty
```

Comment:
Setting value = nil for delete, "" for global and anything else for node doesn't seem very intuitive. 
Is it possible to add a parameter (maybe a type) for the operation {delete, global, node}
```
type opDelete string
type opGlobal string
type opNode string
```
and then that would be an argument you pass in?